src/app/[locale]/(app)/(nav)/(time-selector)/(top)/top/guilds/details/_components/TopChannels.tsx

```diff
@@ -18,7 +18,7 @@ type Channel = Pick<
 
 export default function TopChannels({ channels }: { channels: Channel[] }) {
   return (
-    <Section title="Top channels">
+    <Section title="top.topChannels">
```

Comment:
```suggestion
    <Section title={t("top.topChannels")}>
```
Don't forget to import `t`